src/organizations/components/OrgProfileTab/DeletePanel.tsx

```diff
@@ -67,15 +67,16 @@ const DeletePanel: FC = () => {
     handleDeleteFreeAccount = handleShowWarning
   }
 
-  const showDeleteButtonInFreeAccount = account.type === 'free'
-  const showDeleteButtonInPaidAccount =
-    isFlagEnabled('createDeleteOrgs') &&
-    (account.type === 'pay_as_you_go' || account.type === 'contract')
+  const showDeleteFreeAccountButton =
+    CLOUD && account.type === 'free' && user.orgCount === 1
+
+  const showDeleteOrgButton =
```

Comment:
These boolean variables are easy to confuse with functions. `showDeleteFreeAccountButton()` and `showDeleteOrgButton()` are pretty natural. I think it might be less confusing to name them something like `shouldShowDeleteFreeAccountButton` and `shouldShowDeleteOrgButton`

Reply:
Good point - I'll rename them. 👍 

I need to make a modification to the test for this page too, to cover the intended case - free account with one org in it.